tests/phpunit/Persistence/SitelinkItemPageLookupTest.php

```diff
@@ -29,6 +29,15 @@ protected function setUp(): void {
 		);
 	}
 
+	public function testReturnsNullWhenNoSitelinkSiteIdIsConfigured(): void {
+		$this->assertNull(
+			( new SitelinkPageItemLookup(
+				$this->sitelinkStore,
+				null
+			) )->getItemId( Title::newFromText( 'Foo' ) )
+		);
+	}
```

Comment:
This test isn't great because there are multiple reasons for null to be returned. If you break the behavior under test, you'd still get null.

Reply:
Then again, no need to start cleaning up this test class